shell/platform/embedder/embedder.h

```diff
@@ -141,6 +141,14 @@ typedef enum {
   kFlutterSemanticsFlagHasToggledState = 1 << 16,
   // If true, the semantics node is "on". If false, the semantics node is "off".
   kFlutterSemanticsFlagIsToggled = 1 << 17,
+  // Whether the platform can scroll the semantics node when the user attempts
+  // to move focus to an offscreen child.
+  //
+  // For example, a ListView widget has implicit scrolling so that users can
+  // easily move to the next visible set of children. A TabBar widget does not
```

Comment:
"visible" is easy to misinterpreted here. The focus moves to the currently invisible set of children and makes those visible...

Reply:
Updated.